hkmc2/shared/src/test/mlscript/lifter/Loops.mls

```diff
@@ -27,13 +27,11 @@ fun foo() =
     set i += 1
     fs.push of () => x
 
-// * Note that this also fails without lifting, as we need to fix [fixme:0]
```

Comment:
Pls add a `:dontRewriteWhile` option to reproduce the problem that still exists, instead of trying to sweep it under the rug ;^)